.gitignore

```diff
@@ -1,7 +1,5 @@
 .DS_Store
-.npmignore
 apps/*/translations/en
 node_modules
 public
-acceptance_tests/output/
```

Comment:
Yeah come to think of it, since you changed the tests, I haven't seen it anywhere